pkg/cmd/codespace/list.go

```diff
@@ -34,10 +35,23 @@ func newListCmd(app *App) *cobra.Command {
 		`),
 		Aliases: []string{"ls"},
 		Args:    noArgsConstraint,
-		RunE: func(cmd *cobra.Command, args []string) error {
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			// if repo is provided, we don't accept orgName or userName
+			if opts.repo != "" {
+				return cmdutil.MutuallyExclusive("using `--org` or `--user` with `--repo` is not allowed", opts.repo != "", opts.orgName != "" || opts.userName != "")
+			}
+			if opts.useWeb {
```

Comment:
I don't think this `if` is needed - it's handled by the `MutuallyExclusive` check

Reply:
fixed in [28bbd99](https://github.com/cli/cli/pull/7288/commits/28bbd99bd8948190d8b2fd3532ed014aa0207052)